seqr/views/apis/data_manager_api_tests.py

```diff
@@ -846,8 +915,9 @@ def mock_write(content):
                     warnings = params['extra_warnings'] + warnings
                 deleted_count = params.get('deleted_count', params['initial_model_count'])
                 response_json, new_sample_guid = _test_basic_data_loading(
-                    params['new_data'], params["num_parsed_samples"], 2, 16, body,
-                    '1kg project nåme with uniçøde, Test Reprocessed Project', warnings=warnings,
+                    params['new_data'], params["num_parsed_samples"], params.get('num_loaded_samples', 2), 16, body,
+                    params.get('project_names', '1kg project nåme with uniçøde, Test Reprocessed Project'), warnings=warnings,
```

Comment:
The tests for `splice` has a different number of `num_loaded_samples` and `project_names`.

Reply:
why not just load 2 samples for the splice test though?

Reply:
Good catch. I added `num_loaded_samples` for the `splice` when I was trying to load for two projects. I changed to add two samples to one project in the end. I could change it to add two samples to two projects by configuring the individual id.

Reply:
Now, it always loads two samples to two projects. No more params for `num_loaded_samples` and ` project_names`.